pkg/bff/models/v1/collaborator.go

```diff
@@ -39,3 +40,22 @@ func (collab *Collaborator) Validate() error {
 	// TODO: More comprehensive validation may be required
 	return nil
 }
+
+// Helper to determine if a collaborator invite is valid based on the expiration date.
+// If there is no expiration date, this method assumes that the collaborator invitation
+// is still valid.
```

Comment:
Trying to think through when this would be the case (that there is no expiration) -- is this an edge case for people who registered between the last release and the upcoming release?

Reply:
When a collaborator logs in successfully and "accepts" the invite we clear out the expiration date, which allows them to log in successive times after they have accepted the invite and joined the organization. This means that it's important we clear out the expiration date on login so that we know they are actually a returning user. There are actually a few other ways this could happen though:

1. The user logs in for the first time and a default organization is created for them and they are added as a collaborator - there is no expiration because there was no invite and they are considered an "Organization Leader".
2. A TSP user creates a new organization for a new VASP - similarly to 1 they are added as a collaborator in the new organization and we don't need to send them an invite.